modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -3872,7 +3872,7 @@ a|dbms.security.oidc.<provider>.well_known_discovery_uri, a URI
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|A list of procedures (comma separated) that are to be loaded. The list may contain both fully-qualified procedure names, and partial names with the wildcard `*`. If this setting is left empty no procedures will be loaded.
+a|A list of procedures (comma separated) that are to be loaded. The list may contain both fully-qualified procedure names, and partial names with the wildcard `*`. If this setting is unconfigured, all procedures will be loaded.
```

Comment:
I think it's perhaps worth explaining both scenarios here:

* If this setting is active, but no procedures are included, then no procedures will be loaded.
* If this setting is commented out, and is therefore inactive, all procedures will be loaded.

I'm going to cc @renetapopova, as I know there is a plan to edit the descriptions of the config settings and it may be useful for Reneta to be aware of this change.